client/signup/jetpack-connect/index.jsx

```diff
@@ -136,8 +131,17 @@ class JetpackConnectMain extends Component {
 			this.props.jetpackConnectSite.isFetching;
 	}
 
-	getCurrentUrl() {
-		let url = this.refs.siteUrlInputRef.state.value.toLowerCase();
+	handleUrlChange = ( event ) => {
+		const url = event.target.value;
+		this.setState( {
+			currentUrl: this.cleanUrl( url ),
+			shownUrl: url,
+		} );
+		this.dismissUrl();
```

Comment:
It looks like this is here to cancel a `checkUrl` call when the URL is changed. The order is mostly irrelevant because it's very difficult to reach this condition as the input UI can't be changed while the check is ocurring.

Nonetheless, I'm moving the `dismissUrl` above the state change because the dismiss is supposed to cancel the check when provided with the same URL as is being checked, which is the URL before the change.

@tyxla Seem right?